client/my-sites/stats/stats-module/index.jsx

```diff
@@ -172,16 +172,20 @@ class StatsModule extends Component {
 						title={ this.getModuleLabel() }
 						emptyMessage={ moduleStrings.empty }
 						showMore={
-							this.props.showSummaryLink && displaySummaryLink
+							displaySummaryLink
 								? {
 										url: this.getHref(),
-										label: this.props.translate( 'View all', {
-											context: 'Stats: Button label to expand a panel',
-										} ),
+										label:
```

Comment:
I understand this is by design - it's just a little strange that `View all` doesn't really expand the panel.

Reply:
Agreed, the translator's comment doesn't accurately portray what the link does. Perhaps it should be more along the lines of "Button link to show more detailed stats information".